tests/test_http.py

```diff
@@ -24,6 +25,15 @@ def s():
     return HttpSession(verbose=True)
 
 
+def has_connection():
+    # In case we run tests in an environment without internet connection.
+    try:
+        HttpStream("https://httpbin.org/get", "get")
+        return True
+    except OsbsNetworkException:
+        return False
```

Comment:
Nice! I'd been thinking of `-k not http` for that but this is automatic, much better idea.